odoo/addons/base/models/ir_qweb.py

```diff
@@ -1300,7 +1303,8 @@ def _compile_directive_att(self, el, options, level):
         - value from keys that start with ``t-attf-``: format string
             expression.
         """
-        code = [indent_code("attrs = values['__qweb_attrs__'] = {}", level)]
+        # reset the attributes dictionary
+        code = [indent_code("attrs = values['__qweb_attrs__']", level)]
```

Comment:
And this loads the attributes dictionary (into the locals of the compiled templates), it doesn't reset anything.